pkgdb2client/admin.py

```diff
@@ -413,6 +406,49 @@ def __handle_request_branch(actionid, action):
     return data
 
 
+def __handle_request_unretire(actionid, action):
+    ''' Handle unretirement requests. Do the same checks as done for new
+    package requests and ask the admin to do the necessary steps by hand. '''
+
+    bugid = action['info']['pkg_review_url'].rsplit('/', 1)[1]
+    if '=' in bugid:
+        bugid = bugid.split('=', 1)[1]
+
+    # Add valuees to info that pkgdb adds to new package actions
+    action['info']['pkg_name'] = action['package']['name']
+    action['info']['pkg_summary'] = action['package']['summary']
+    action['info']['pkg_collection'] = action['collection'][
+        'branchname']
+    action['info']['pkg_poc'] = action['user']
+    #  FIXME utils.check_package_creation() also tries to get
+    #  pkg_namespace, but this does not seemt to be present in the
+    #  action object
```

Comment:
Should we include: ``action['package'].get('namespace', 'rpms')`` already so that when pkgdb2 is fixed, this will be as well?